test/fixtures/other/spread-attributes/input.astro

```diff
@@ -0,0 +1,5 @@
+<div {...{datatype:      ""}}></div>
+
+<div {...myObject}></div>
+
+<div {...get({ thing: 'hey'})}></div>
```

Comment:
In that specific case, it also work because a string is a valid JavaScript value, though it won't do anything to the result (and in the editor you get an error telling you to spread from an object). Same thing with other valid values

If the value is really invalid, you'll get an error with an helpful relevant message, ex: 
```
[error] src/pages/index.astro: SyntaxError: Binding should be performed on object property. (1:6)
[error] > 1 | <>{::Fsdfsdfs
[error]     |      ^
[error]   2 | }</>
```